mesh/mesh.sh

```diff
@@ -1,6 +1,7 @@
 #!/usr/bin/env bash
 
-source env.sh
+#source env.sh #custom configuration (i.e. OpentTest)
+source fake-mesh-env.sh
```

Comment:
leave as fake-mesh-example-env.sh. README says one of the examples should be copied to env.sh (which is .gitignored). Otherwise we'd need to change the source to switch between fake-mesh and opentest